src/index.html

```diff
@@ -59,33 +59,360 @@
     </header>
 
     <main class="catalog">
+
       <div class="catalog__card">
-        Product card
+
+        <img 
+            src="./images/imac.jpeg" 
+            alt="Samsung Galaxy S22 2022 8/128GB Green" 
+            class="catalog__img"
+        >
```

Comment:
Identation must be 2 spaces
```suggestion
        <img 
          src="./images/imac.jpeg" 
          alt="Samsung Galaxy S22 2022 8/128GB Green" 
          class="catalog__img"
        >
```